src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -278,10 +366,9 @@ bool KeyDropDownControl::CheckRepeatedModifier(StackPanel parent, uint32_t dropD
     return matchPreviousModifier;
 }
 
-// Function to set the flyout warning message
-void KeyDropDownControl::SetDropDownError(ComboBox dropDown, TextBlock messageBlock, hstring message)
+// Function to set the warning message
+void KeyDropDownControl::SetDropDownError(ComboBox currentDropDown, hstring message)
 {
-    messageBlock.Text(message);
-    dropDown.ContextFlyout().ShowAttachedFlyout((FrameworkElement)dropDown);
-    dropDown.SelectedIndex(-1);
+    currentDropDown.SelectedIndex(-1);
```

Comment:
I think it would be a better user experience to display the Shortcut as it was selected by the user, rather than deleting the conflicting key. Then the user can chose to resolve the conflict as it sees fit. I find it kind of jarring to have the shortcut as displayed in the dropdowns not match what I just typed.

Reply:
As discussed offline, we can make this change later in a separate PR, which might be easier with bindings.